### DISCUSSION TITLE
Time to roll a new PRQL version - 0.3.2?

### DISCUSSION THREAD

**Discussion by richb-hanover** [PRQL/prql#1360] [General]
There are lots of cool features (`switch`, `in`, `union`, others) in the compiler now.

Is it worth rolling out a new version (say, 0.3.2) so that it can be noted in "official" CHANGELOG, and also in a new VScode extension?

(Or are there cool things imminent that warrant waiting a bit?) Thanks.

**Comment by max-sixty** [comment]
Yes, let's discuss on the next dev call when we do 0.4.0.

We were planning to combine it with a marketing push, e.g. HackerNews. But that's less viable now that we recently hit the top of HackerNews again organically.

So I would vote to do it quite soon if others agree. Here's the milestone: https://github.com/PRQL/prql/milestone/4

**Reply by richb-hanover** [reply]
> Yes, let's discuss on the next dev call when we do 0.4.0.

Would you push toward 0.4.0 instead of shipping a 0.3.2 substantially "as-is"? From a selfish standpoint, the `switch` statement is very interesting, and a 0.3.2 version might cause you to roll out a new VScode extension that implements it :-)

As always, I advocate, you decide ...

**Reply by max-sixty** [reply]
There are some breaking changes, so we should bump to 0.4.0, but we can do that without much delay I think! Gotta finish the docs & changelog...

**Comment by richb-hanover** [comment]
PS When's the next Dev Call? (I'll try to make it.) Maybe you could update https://github.com/PRQL/prql/issues/1083

**Reply by max-sixty** [reply]
Great, updated #1083!
